Changes for windows

satyahbp opened the pull request:
- In setup.py, added encoding as "utf-8" while opening README.md, without which setup was not working for windows.
- Added a temporary message in panther > cli > main.py for shell( ), saying "Currently this feature is not supported for Windows", can be removed when the feature is added.

AliRn76 commented on panther/cli/main.py:
```diff
@@ -11,7 +11,10 @@
 
 def shell() -> None:
-    os.system('bpython')
+    if sys.platform == "win32":
```
```python
platform.system() == 'Windows'
```
is more reliable than `sys.platform`

satyahbp commented on panther/cli/main.py:
```diff
@@ -11,7 +11,10 @@
 def shell() -> None:
-    os.system('bpython')
+    if sys.platform == "win32":
+        cli_error("Currently this feature is not supported for Windows.")
```
Ok.

satyahbp replied:
I'll make changes accordingly

AliRn76 commented:
Please upgrade the `__version__` in `panther/__init__.py` and 
Add the changes in `docs/docs/release_notes.md`

satyahbp commented:
Updated as mentioned.
Added another condition in cli, to check if user is sending any arguments. 
Earlier it was showing error when "panther" command is typed without any argument.
